Add support for multiple (named) return parameters in Natspec devdoc (#7534)
ethereum/solidity: Changelog.md

 * Standard JSON Interface: Add option to disable or choose hash method between IPFS and Swarm for the bytecode metadata.
 * Syntax: ``push(element)`` for dynamic storage arrays do not return the new length anymore.
 * Type checker: Resulting type of exponentiation is equal to the type of the base. Also allow signed types for the base.

>  * Natspec JSON Interface: Support multiple ``@return`` statements in dev documentation to behave like named parameters.

Reviewer: I think we need to be a little bit more specific and mention both, the fixed support and the breaking implication of multiple return statements in the dev docs.

```suggestion
 * Natspec JSON Interface: Support multiple ``@return`` statements in ``@dev`` documentation and force them to have define a name if parameter is named.
```

Language Features:

Compiler Features:

Bugfixes:

> ### 0.5.13 (unreleased)

Reviewer: There's an additional headline that slipped down there: https://github.com/ethereum/solidity/pull/7534/files#diff-9b8fab691c00b9e5380b19ce882f3271R44.

 * Syntax: ``push(element)`` for dynamic storage arrays do not return the new length anymore.
 * Inline Assembly: Only strict inline assembly is allowed.
 * Type checker: Resulting type of exponentiation is equal to the type of the base. Also allow signed types for the base.

>  * Natspec JSON Interface: Support multiple ``@return`` statements in dev documentation to behave like parameters where the first word in the entry must contain the name.

Reviewer: ```suggestion
 * Natspec JSON Interface: Properly support multiple ``@return`` statements in ``@dev`` documentation and enforce named return parameters to be mentioned documentation.
```
